server_nodes.tf

```
@@ -91,7 +106,7 @@ locals {
         [
           "--node-ip ${server.ip}",
           "--node-name '${try(server.name, key)}'",
-          "--server https://${local.root_server_ip}:6443",
+          "--server https://${local.root_advertise_ip_curl}:6443",
```

Updated variable name.

```suggestion
          "--server https://${local.root_advertise_ip_k3s}:6443",
```